piccolo_api/change_password/endpoints.py

```diff
@@ -192,5 +232,7 @@ class _ChangePasswordEndpoint(ChangePasswordEndpoint):
         _login_url = login_url
         _change_password_template = change_password_template
         _styles = styles or Styles()
+        _session_table = session_table

```

Comment:
Thanks, I'll fix it.

Reply:
Reminder. You forgot to add this line.

Reply:
It seems to work for me - do you still get an error?

Reply:
In Piccolo admin, can we just pass in `SessionsBase`?

```python
change_password(session_table = SessionsBase)
```

We could add `SessionsBase` as a default, but my only concern is it ties this endpoint to session auth. I was hoping it could be used with other forms of auth too, but maybe it's unrealistic.

Reply:
You are right. Now I tried `change_password(session_table=SessionsBase)` in Piccolo Admin and everything work fine. Sorry.

Reply:
It's fine. I thought about it, and I think you're right - let's just make `SessionsBase` the default. If someone doesn't want to use it, they just set it to `None`. I think most people will use it with `SessionsBase` any way, so it's a sensible default.

What I'd like to add in a future PR is hooks, so people can perform their own logic when the password change is successful. That way people might be able to use it with other forms of auth.